src/gfx/math.hpp

```diff
@@ -40,6 +40,11 @@ template <size_t Alignment> inline constexpr size_t alignTo(size_t size) {
   }
 }
 
+template <typename T> inline bool isRoughlyEqual(T val, T target, T tolerance = 0.05f) {
```

Comment:
typename std::enable_if<std::is_floating_point<T>::value, or another number type where you can assign 0.05f; or remove the default value